### ISSUE TITLE
All-atom SBM Hamiltonian

### ISSUE THREAD

**Issue by ajkluber** [ajkluber/model_builder#16]
I am going to start development of an all-atom Structure-based model Hamiltonian on the `devel` branch. This shouldn't interfere with other branches but we should be conscious to merge branches whenever possible.

**Comment by TensorDuck** [comment]
I can merge the awsem test branch now if you would like. Right now, you can see the differences are only on four files:
https://github.com/ajkluber/model_builder/compare/test_awsem

I just need to add back some lines I commented out for debugging purposes. And I need to handle the debye stuff and sourcing + parmaeterizing the model better, per our discussion in the debye thread. I think that would definitely be completely separate.

**Comment by ajkluber** [comment]
Yeah, please merge whenever you are able. The work will probably be separate but I might need to edit `inputs.py` at some point.

**Comment by TensorDuck** [comment]
merged my branch with master. I'll continue working on the paramterization of the awsem hamiltonian and will merge that when I'm done, but that should only affect awsem files.
